src/Pages/Login/Login.js

```diff
@@ -21,15 +21,14 @@ class Login extends React.Component {
             async function(){
                 var user = fire.auth().currentUser;
                 if(user.emailVerified === false){
```

Comment:
Don't use var.

Also, create auth instance in firebase file and use it here, no need to call fire.auth() again and again.